contrib/caldav/config.sample

```diff
@@ -18,6 +18,13 @@ Path = /path/to/calendar/on/the/server/
 # Enable this if you want to skip SSL certificate checks.
 InsecureSSL = No
 
+# This option allows you to filter the type of tasks synced and the valid options are
+# event,apt, recur-event, recur-apt, todo, recur, cal exactly like the command line option
```

Comment:
Nit: Missing space after `event,`. Maybe also mention that there should not be any whitespace in the comma-separated list itself.